test/performance/odp_ipsecfwd.c

```diff
@@ -365,14 +365,37 @@ static void print_usage(void)
 	       "Simple IPsec performance tester. Forward and process plain and ipsec packets.\n"
 	       "\n"
 	       "Examples:\n"
-	       "    %s -i ens9f1 -s /etc/odp/sa.conf -f /etc/odp/fwd.conf\n"
+	       "    %s -i ens9f1 -C /etc/odp/ipsecfwd.conf\n"
 	       "\n"
-	       "    With sa.conf containing, for example:\n"
-	       "        0 222 192.168.1.10 192.168.1.16 4 jWnZr4t7w!zwC*F- 0 2"
-	       " n2r5u7x!A%%D*G-KaPdSg 0 12\n"
+	       "    With ipsecfwd.conf containing, for example:\n"
+	       "        default: {\n"
+	       "            sa_dir = 1\n"
+	       "            sa_proto = 0\n"
+	       "            sa_mode = 0\n"
+	       "            sa_crypto: {\n"
+	       "                cipher_alg = 4\n"
+	       "                cipher_key = \"jWnZr4t7w!zwC*F-\"\n"
+	       "                auth_alg = 2\n"
+	       "                auth_key = \"n2r5u7x!A%%D*\"\n"
+	       "                icv_len = 12\n"
+	       "            };\n"
+	       "        };\n"
 	       "\n"
-	       "    With fwd.conf containing, for example:\n"
-	       "        192.168.1.0/24 ens9f1 aa:bb:cc:dd:11:22\n"
+	       "        sa0: {\n"
+	       "            sa_spi = 1337\n"
+	       "            sa_out: {\n"
+	       "                tunnel: {\n"
+	       "                    src_addr = \"192.168.1.10\"\n"
+	       "                    dst_addr = \"192.168.1.16\"\n"
+	       "                };\n"
+	       "            };\n"
+	       "        };\n"
+	       "\n"
+	       "        fwd0: {\n"
+	       "            prefix: \"192.168.1.16/24\"\n"
```

Comment:
I guess this should be 192.168.1.0/24 unless the non-prefix part of the address has some signifigance here which I do not think it has. And the same comment for the config file example.

Reply:
Copy paste error, no significance, fixed in v3.